pkgs/by-name/pa/passes/package.nix

```
@@ -29,7 +29,7 @@ stdenv.mkDerivation (finalAttrs: {
 
   postPatch = ''
     substituteInPlace src/model/meson.build \
-      --replace-fail /app/lib ${zint}/lib
+      --replace-fail /app/lib ${zint.lib}/lib
```

We should probably use `getLib` in case the output gets changed again

```suggestion
      --replace-fail /app/lib ${lib.getLib zint}/lib
```